pkg/broker/server/sse.go

```diff
@@ -167,7 +181,7 @@ func (b *Broker) run() {
 			// Disconnect all clients
 			b.clients.Walk(
 				func(key string, value interface{}) bool {
-					chset, ok := value.(map[chan []byte]int)
+					chset, ok := value.(map[chan []byte]clientValue)
```

Comment:
can't we just have a simpler value in the radix tree without the type `clientValue`?  

We really just need something like `map[chan[]byte]bool`, where the value bool is exactMatch or not.